codecov.yml

```diff
@@ -1,9 +1,9 @@
 fixes:
   - "ballerina/http/**/::../ballerina/"
+  - "io/ballerina/stdlib/http/compiler/::./compiler-plugin/src/main/java/io/ballerina/stdlib/http/compiler/"
 
 ignore:
   - "ballerina-tests"
-  - "compiler-plugin-tests"
```

Comment:
Is there a reason to remove this?

Reply:
I followed [the sample PR given with the issue](https://github.com/ballerina-platform/module-ballerina-graphql/pull/1288/files#diff-3e878d9bdc47f29a0ab909a7db939acf08d2d2f1aaba81e6f897b32361fa40dbL6). Since this has no source files, this does not matter right? Or should I revert this change?

Reply:
> Since this has no source files, this does not matter right?

Yeah. No harm removing it.